x-pack/plugins/maps/server/register_integrations.ts

```diff
@@ -35,4 +35,49 @@ export function registerIntegrations(
     shipper: 'other',
     isBeta: false,
   });
+  customIntegrations.registerCustomIntegration({
+    id: 'ingest_geojson',
+    title: i18n.translate('xpack.maps.registerIntegrations.geojson.integrationTitle', {
+      defaultMessage: 'GeoJson',
+    }),
+    description: i18n.translate('xpack.maps.registerIntegrations.geojson.integrationDescription', {
+      defaultMessage: 'Upload GeoJson with Elastic Maps.',
+    }),
+    uiInternalPath: core.http.basePath.prepend(
+      getFullPath(`?${OPEN_LAYER_WIZARD}=${WIZARD_ID.GEO_FILE}`)
+    ),
+    icons: [
+      {
+        type: 'eui',
+        src: 'logoMaps',
+      },
+    ],
+    categories: ['upload_file', 'geo'],
+    shipper: 'other',
+    isBeta: false,
+  });
+  customIntegrations.registerCustomIntegration({
+    id: 'ingest_shape',
+    title: i18n.translate('xpack.maps.registerIntegrations.shapefile.integrationTitle', {
+      defaultMessage: 'Shapefile',
+    }),
+    description: i18n.translate(
+      'xpack.maps.registerIntegrations.shapefile.integrationDescription',
+      {
+        defaultMessage: 'Upload Shapefile with Elastic Maps.',
```

Comment:
@gchaps Is this the correct wording? Is "Elastic Maps" correct? Should it just be "Maps"?